students/johnn/mailroom/mailroom/ui.py

```diff
@@ -0,0 +1,250 @@
+#!/usr/bin/env python3
+
+import sys
+from pprint import pprint
+from mailroom.donors import Donors, Donor, save_donor_file
```

Comment:
you might want to try "relative imports" here:

`from .donors import Donors, ...`